os-autoinst-obs-auto-submit

```diff
@@ -102,16 +150,13 @@ update_package() {
     rc=0
     wait_for_package_build "$submit_target"
     for target in "${targets[@]}"; do
-        cmd="$osc sr"
+        [[ $target == none ]] && continue # allow specifying submit_target_extra=none for easier testing
         log-info "## Ready to submit $package to $target ##"
-        req=$(get_obs_sr_id "$target" "$dst_project" "$package" || :)
-        # TODO: check if it's a different revision than HEAD
-        if test -n "$req"; then
-            cmd="$cmd -s $req"
-        fi
-        if ! $cmd -m "Update to ${version}" "$target"; then
-            rc=$?
-            failed_packages+=("$package:$rc")
+        local git_branch=${git_branches[$target]}
+        if [[ $git_branch ]]; then
```

Comment:
With this change it's a bit obscurednow  where `rc` gets set. Maybe the `make_...request` functions could actually return `$rc` and it can be catched here? Not sure

Reply:
And actually, you are setting `local rc` in `make_obs_submit_request` (but not in `make_git_submit_request`), so it won't change the outer `rc` here